petpals/messaging/templates/conversation.html

```diff
@@ -0,0 +1,55 @@
+{% extends '_layout.html' %}
+{% block title %}Messages{% endblock %}
+{% set active_page = 'messages' %}
+
+{% block body %}
+
+        <meta id="conversation" data-sender="{{sender}}" data-conversation_id="{{conversation_id}}" data-recipient="{{recipient}}" data-current_user="{{current_user.username}}">
+
+        <section class="bg-white p-2">
```

Comment:
`bg-white` isn't doing anything, nor should it be there in the first place.

Reply:
Removed!